### PR TITLE
Fix line length offenders in the o.e.search package.

### PR DESCRIPTION
**Pull request by jpountz**
Relates #34884

### CODE REVIEW

File: server/src/main/java/org/elasticsearch/search/aggregations/metrics/HyperLogLogPlusPlus.java [hunk 1]
```diff
@@ -86,74 +81,675 @@ public static long memoryUsage(int precision) {
...
+                    184.4114, 186.033, 188.5124, 189.5564, 191.6008, 192.4172, 193.8044, 194.997, 197.4548, 198.8948, 200.2346, 202.3086,
+                    203.1548, 204.8842, 206.6508, 206.6772, 209.7254, 210.4752, 212.7228, 214.6614, 215.1676, 217.793, 218.0006, 219.9052,
+                    221.66, 223.5588, 225.1636, 225.6882, 227.7126, 229.4502, 231.1978, 232.9756, 233.1654, 236.727, 238.1974, 237.7474,
+                    241.1346, 242.3048, 244.1948, 245.3134, 246.879, 249.1204, 249.853, 252.6792, 253.857, 254.4486, 257.2362, 257.9534,
+                    260.0286, 260.5632, 262.663, 264.723, 265.7566, 267.2566, 267.1624, 270.62, 272.8216, 273.2166, 275.2056, 276.2202,
+                    278.3726, 280.3344, 281.9284, 283.9728, 284.1924, 286.4872, 287.587, 289.807, 291.1206, 292.769, 294.8708, 296.665,
+                    297.1182, 299.4012, 300.6352, 302.1354, 304.1756, 306.1606, 307.3462, 308.5214, 309.4134, 310.8352, 313.9684, 315.837,
+                    316.7796, 318.9858, },
+            // precision 7
+            { 92, 93.4934, 94.9758, 96.4574, 97.9718, 99.4954, 101.5302, 103.0756, 104.6374, 106.1782, 107.7888, 109.9522, 111.592,
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/search/aggregations/metrics/HyperLogLogPlusPlus.java]: hahahahahhaahahha

File: server/src/main/java/org/elasticsearch/search/aggregations/metrics/HyperLogLogPlusPlus.java [hunk 2]
```diff
@@ -454,14 +1049,12 @@ public int hashCode(long bucket) {
     }
 
     public boolean equals(long bucket, HyperLogLogPlusPlus other) {
-        return Objects.equals(p, other.p) &&
-                Objects.equals(algorithm.get(bucket), other.algorithm.get(bucket)) &&
-                Objects.equals(getComparableData(bucket), other.getComparableData(bucket));
+        return Objects.equals(p, other.p) && Objects.equals(algorithm.get(bucket), other.algorithm.get(bucket))
```
Comment (nik9000) [inline server/src/main/java/org/elasticsearch/search/aggregations/metrics/HyperLogLogPlusPlus.java]: Could you do these one per line?

File: server/src/test/java/org/elasticsearch/search/aggregations/bucket/ShardSizeTestCase.java [hunk 3]
```diff
@@ -90,7 +90,8 @@ protected void indexData() throws Exception {
 
         indexRandom(true, docs);
 
-        SearchResponse resp = client().prepareSearch("idx").setTypes("type").setRouting(routing1).setQuery(matchAllQuery()).execute().actionGet();
+        SearchResponse resp = client().prepareSearch("idx").setTypes("type").setRouting(routing1).setQuery(matchAllQuery()).execute()
```
Comment (nik9000) [inline server/src/test/java/org/elasticsearch/search/aggregations/bucket/ShardSizeTestCase.java]: Maybe just use `get`?

File: server/src/test/java/org/elasticsearch/search/basic/SearchWithRandomIOExceptionsIT.java [hunk 4]
```diff
@@ -150,17 +153,21 @@ public void testRandomDirectoryIOExceptions() throws IOException, InterruptedExc
...
         final int numSearches = scaledRandomIntBetween(10, 20);
         // we don't check anything here really just making sure we don't leave any open files or a broken index behind.
         for (int i = 0; i < numSearches; i++) {
             try {
                 int docToQuery = between(0, numDocs - 1);
                 int expectedResults = added[docToQuery] ? 1 : 0;
                 logger.info("Searching for [test:{}]", English.intToEnglish(docToQuery));
-                SearchResponse searchResponse = client().prepareSearch().setTypes("type").setQuery(QueryBuilders.matchQuery("test", English.intToEnglish(docToQuery)))
+                SearchResponse searchResponse = client().prepareSearch().setTypes("type")
+                        .setQuery(QueryBuilders.matchQuery("test", English.intToEnglish(docToQuery)))
```
Comment (nik9000) [inline server/src/test/java/org/elasticsearch/search/basic/SearchWithRandomIOExceptionsIT.java]: I think this indentation doesn't line up with the line under it which'll be confusing to read.

File: server/src/test/java/org/elasticsearch/search/geo/GeoBoundingBoxIT.java [hunk 5]
```diff
@@ -225,7 +229,8 @@ public void testCompleteLonRange() throws Exception {
         assertThat(searchResponse.getHits().getTotalHits(), equalTo(1L));
         searchResponse = client().prepareSearch()
                 .setQuery(
-                        geoBoundingBoxQuery("location").setValidationMethod(GeoValidationMethod.COERCE).setCorners(50, 0, -50, 360).type("indexed")
+                        geoBoundingBoxQuery("location").setValidationMethod(GeoValidationMethod.COERCE).setCorners(50, 0, -50, 360)
+                        .type("indexed")
```
Comment (nik9000) [inline server/src/test/java/org/elasticsearch/search/geo/GeoBoundingBoxIT.java]: Having this line on the same indention of as the one that builds the object that it is poking doesn't quite feel right to me.

File: server/src/test/java/org/elasticsearch/search/scroll/SearchScrollIT.java [hunk 6]
```diff
@@ -349,13 +370,15 @@ public void testClearIllegalScrollId() throws Exception {
...
+        client().admin().indices().prepareCreate("test").setSettings(Settings.builder().put("index.number_of_shards", 3)).execute()
+        .actionGet();
         client().admin().cluster().prepareHealth().setWaitForEvents(Priority.LANGUID).setWaitForGreenStatus().execute().actionGet();
 
         client().admin().cluster().prepareHealth().setWaitForEvents(Priority.LANGUID).setWaitForGreenStatus().execute().actionGet();
 
         for (int i = 0; i < 100; i++) {
-            client().prepareIndex("test", "type1", Integer.toString(i)).setSource(jsonBuilder().startObject().field("field", i).endObject()).execute().actionGet();
+            client().prepareIndex("test", "type1", Integer.toString(i)).setSource(jsonBuilder().startObject().field("field", i).endObject())
+            .execute().actionGet();
```
Comment (nik9000) [inline server/src/test/java/org/elasticsearch/search/scroll/SearchScrollIT.java]: This one doesn't look right.

File: server/src/test/java/org/elasticsearch/search/suggest/CompletionSuggestSearchIT.java [hunk 7]
```diff
@@ -1027,7 +1053,8 @@ public void testPrunedSegments() throws IOException {
                 .field("somefield", "somevalue")
                 .endObject()
         ).get(); // we have 2 docs in a segment...
-        ForceMergeResponse actionGet = client().admin().indices().prepareForceMerge().setFlush(true).setMaxNumSegments(1).execute().actionGet();
+        ForceMergeResponse actionGet = client().admin().indices().prepareForceMerge().setFlush(true).setMaxNumSegments(1).execute()
```
Comment (nik9000) [inline server/src/test/java/org/elasticsearch/search/suggest/CompletionSuggestSearchIT.java]: Maybe just call `get`?

**Review by nik9000 — APPROVED** [review]
LGTM. Thanks for all that cleaning!

**Comment by nik9000** [comment]
I'm sorry for your merge conflicts!

**Comment by jpountz** [comment]
No worries, that shouldn't be too hard to fix. Thanks for reviewing!

**Comment by jpountz** [comment]
I am not backporting this one to 6.x as there is a number of conflicts that make this change unsafe. An alternative would be to re-do the change on 6.x but it is very time-consuming.

**Pull request merged**